.ci/visual-regression-test.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+# Variables
+BUILD_DIR=$(pwd)
+GITHUB_API_URL="https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME"
+
+# Check if we are NOT on the master branch and this is a PR
+if [[ ${CURRENT_BRANCH} != "master" && -z ${CI_PR_URL} ]];
+then
+  echo -e "\nVisual regression tests will only run if we are not on the master branch and making a pull request"
+  exit 0;
+fi
+
+echo -e "\nProcessing pull request #$PR_NUMBER"
+
+LAST_GIT_COMMIT_MESSAGE=$(git log -1 --pretty=%B)
```

Comment:
It would be nifty if the code here could be written into a Terminus plugin, and called e.g. via `terminus visual-regression:check-needed`.  Default behavior could be as written, with options to control files needed / files not needed / special string in comment, etc.

Reply:
I agree but that seems out of scope for an MVP. Also, since we are showing this off in the Drupalcon training I would rather be verbose with the logic so folks can see what is going on rather than hiding it behind a Terminus command.